File: eFrizer/eFrizer.Model/Service/Service.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace eFrizer.Model
{
    public class Service
    {
        public int ServiceId { get; set; }
        public string Name { get; set; }
        public string Description { get; set; }

        public float Price { get; set; }

        public int TimeMin { get; set; }
    }
}

File: eFrizer/flutter_efrizer/lib/models/service.dart
import 'dart:ffi';
import 'package:flutter/material.dart';


class Service{
  int? ServiceId;
  String? Name;
  String? Description;
  int Price;
  int? TimeMin;

  Service({
    this.ServiceId,
    this.Name,
    this.Description,
    required this.Price,
    this.TimeMin,
  });

  factory Service.fromJson(Map<String, dynamic> json) {
    return Service(
      ServiceId: int.parse(json['serviceId'].toString()),
      Name: json['name'],
      Description: json['description'],
      Price: int.parse(json['price'].toString()),
      TimeMin: int.parse(json['timeMin'].toString()),
    );
  }

  Map<String, dynamic> toJson() => {
        
      };

}